python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -386,1649 +740,3318 @@ def GetTransform(self, targetname, connectedBodyName='', linkName='', geometryNa
+        if toolname is not None:
+            taskparameters['toolname'] = toolname
+        if robotBridgeConnectionInfo is not None:
+            taskparameters['robotBridgeConnectionInfo'] = robotBridgeConnectionInfo
+        if locationCollisionInfos is not None:
+            taskparameters['locationCollisionInfos'] = locationCollisionInfos
+        if robotspeed is not None:
+            taskparameters['robotspeed'] = robotspeed
+        if robotaccelmult is not None:
+            taskparameters['robotaccelmult'] = robotaccelmult
+        if ionames is not None:
+            taskparameters['ionames'] = ionames
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def GetMotorControlParameterSchema(self, timeout=10, **kwargs):
-        """Gets motor control parameter schema
+    def GetMotorControlParameterSchema(self, timeout=10, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
+        # type: (float, Optional[types.GetMotorControlParameterSchemaParametersDynamicEnvironmentState], Optional[int], Optional[Any]) -> Optional[Any]
+        """
+        Gets motor control parameter schema
 
         Args:
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
         """
         taskparameters = {
             'command': 'GetMotorControlParameterSchema',
-        }
+        }  # type: dict[str, Any]
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def GetMotorControlParameter(self, jointName, parameterName, timeout=10, **kwargs):
-        """Gets motor control parameters as a name-value dict, e.g.: {'J1':{'KP':1}, 'J2':{'KV':2}}
+    def GetMotorControlParameter(self, jointName, parameterName, timeout=10, dynamicEnvironmentState=None, debuglevel=None, unit='mm', robotname=None, toolname=None, robotBridgeConnectionInfo=None, locationCollisionInfos=None, robotspeed=None, speed=_deprecated, robotaccelmult=None, ionames=None, **kwargs):
+        # type: (str, str, float, Optional[types.GetMotorControlParameterParametersDynamicEnvironmentState], Optional[int], str, Optional[str], Optional[str], Optional[types.GetMotorControlParameterParametersRobotBridgeConnectionInfo], Optional[list[types.GetMotorControlParame
```

Comment:
parameters changed

Reply:
:facepalm: forgot to add the generator settings.

https://github.com/mujin/mujinplanningclientpy/pull/45/commits/03baf45366bc9b8133360a26787d83a2863eb53e